seqr/management/tests/lift_variant_to_hg38_2_3_tests.py

```diff
@@ -7,7 +7,7 @@
 from django.core.management import call_command
 from django.test import TestCase
 from django.core.management.base import CommandError
-from seqr.utils.elasticsearch.es_utils_tests import PARSED_VARIANTS
+from seqr.utils.elasticsearch.es_utils_2_3_tests import PARSED_VARIANTS
```

Comment:
This test uses a constant defined by the elasticsearch utils test. I'm not sure whether we should move the constant to a common test util file.

Reply:
yes we should move the constants to the main test_utils file where the other test constants live

Reply:
Moved PARSED_VARIANTS and PARSED_SV_VARIANTS to seqr.views.utils.test_utils.